features/step_definitions/navigation_steps.rb

```diff
@@ -302,6 +302,16 @@
   find_link('Configuration').click
 end
 
+Given(/^I am on the Configuration => Systems => Target systems page$/) do
+  step %(I am authorized)
+  visit("https://#{$server.full_hostname}/rhn/configuration/system/TargetSystems.do")
+end
+
+Given(/^I type "sle-minion" in the search box and click "Go"$/) do
```

Comment:
we are hardcoding a parameter here. ( you need a general one step here) where we pass the regex and put them in a var
https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/navigation_steps.rb#L220

Reply:
additionaly, you could even make the 'GO" click a parameter.